api/api-users-v3/src/main/java/com/thoughtworks/go/apiv3/users/representers/UserRepresenter.java

```diff
@@ -33,6 +33,7 @@ public static void represent(OutputWriter childWriter, UserToRepresent user) {
                 .add("email", user.getEmail())
                 .add("email_me", user.isEmailMe())
                 .add("is_admin", user.isAdmin())
+                .add("is_individual_admin", user.isIndividualAdmin())
```

Comment:
@vrushaliwaykole -- From the implementation perspective of the SPA, it might make sense to add this field to the user response, but not sure if from the Users API consumer perspective if it adds any value.

A consumer of the API will want to know whether the user is an admin or not, regardless whether the user is admin directly or via a role.

---

To check if the user is an admin directly or via a role, we can rely upon the `super admin`'s  API at the client (browser) side.


##### Implementation details:

On the page load, we can make a call to users API and roles API and then construct the Users Model based on the responses of both.